app/views/ops/_schedule_form_timer.html.haml

```diff
@@ -43,6 +43,7 @@
                    :name             => "start_date",
                    "ng-model"        => "scheduleModel.start_date",
                    "datepicker-init" => "",
+                   "miq-calendar"    => true,
```

Comment:
Yes, I originally wanted to merge 'datepicker-init' and 'miq-calendar' into one directive, but I could not do it, because
the controllers hooked to retire & schedule forms work with the dates differently: one uses `Date` objects (and
relies on parser & formatter, the other one uses plain strings.

We can use both 'datepicker-init` and 'miq-calendar' at the same time, as long as we use `Date` objects in
the model and `Date` is the preferred data type, am I correct?

I do realize, that since I'm doing this, I could also get rid of `start_date`, `start_hour`, `start_min`, and `time_zone`
in `scheduleFormController` and use just one object containing all the information we need. This can also be done
in this PR, is that's what's preferred. But I don't want to be splitting `miq-calendar` into two directives, because
then instead of two directives, we'd have three :-(